### PR TITLE
6.5 : starboard upgrade to 0.15.4

### PR DESCRIPTION
**Pull request by anupamtamrakar**
6.5 Deployment changes for starboard upgrade.

cc @deven0t

### CODE REVIEW

**Comment by semyonmor** [comment]
Waiting for approval from Dev Team

**Review by deven0t — APPROVED** [review]

**Review by KoppulaRajender — APPROVED** [review]

**Pull request merged**
